build/package/i2b2/sql/50-stored-procedures/get_ontology_elements.plpgsql

```diff
@@ -0,0 +1,35 @@
+-- pl/pgsql function that returns (maximum lim) ontology elements whose paths contain the given search_string.
+
+CREATE OR REPLACE FUNCTION i2b2metadata.get_ontology_elements(search_string varchar, lim integer DEFAULT 10)
```

Comment:
Indeed, actually none of the functions in `50-stored-procedures` are used or up to date at the moment. I've included them though in anticipation to the next implementations, but it is likely that it will be needed for them to be modified.

I can remove for clarity if you prefer, or leave them here, maybe with a readme saying what I just wrote.

Reply:
Fine, don't touch them, I'll take care of it.